db_connect.js

```diff
@@ -1,4 +1,4 @@
-// ********** Connection to database ************
+// *********** Connection to database ************
```

Comment:
for your credentials, you could consider using and environment file as opposed to hard coding.
you should export a singleton instance of the connection as when this file is required anytime it will create a new connection object.